pkgs/top-level/release-small.nix

```diff
@@ -2,7 +2,7 @@
    the load on Hydra when testing the `stdenv-updates' branch. */
 
 { nixpkgs ? { outPath = (import ../../lib).cleanSource ../..; revCount = 1234; shortRev = "abcdef"; }
-, supportedSystems ? [ "x86_64-linux" "i686-linux" "x86_64-darwin" ]
+, supportedSystems ? [ "x86_64-linux" "x86_64-darwin" ]
```

Comment:
Wasn't the plan to build the nixos-small channel for `i686-linux` but maybe remove it as constituents of the tested job?